lib/capybara/spec/session/assert_text_spec.rb

```diff
@@ -15,6 +15,19 @@
     expect(@session.assert_text('text with whitespace', normalize_ws: true)).to eq(true)
   end
 
+  context 'with enabled default collapsing whitespace' do
+    before do
+      @previous_default_normalize_ws = Capybara.default_normalize_ws
```

Comment:
Since `normalize_ws` is reset in spec_helper there's no need to store/restore the previous